src/redux/createState.js

```diff
@@ -0,0 +1,12 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger, thunk));
+
+export default store;
```

Comment:
- [ ] Kindly remove all these codes as they are not required and are not part of this project. For this project, you're required to set up and initialize components for the Bookstore app. The disadvantage of having this redux code and functionalities is that your project will be seen as copied code from someone else. So I highly recommend you remove all redux features for now till you're asked to add them.